### ISSUE TITLE
Update submit transaction view designs

### ISSUE THREAD

**Issue by erin-hughes** [IBM-Blockchain/blockchain-vscode-extension#2639]
Now that we're revisiting the transaction view, we should update the designs to better fit in with the changes that have been made in the extension since then.

The most recent designs that were completed for this view are [here](https://user-images.githubusercontent.com/52786879/69735178-32fc9700-1128-11ea-8c86-982f04c2d09e.png) and [here](https://user-images.githubusercontent.com/52786879/69735189-36901e00-1128-11ea-924e-c8f086a583b1.png). Originally we had two pages: a home page that included a log of recently submitted transactions, and a form that users could fill in to submit their transactions. We dropped the idea of the home page and focused solely on the form aspect, before pausing on implementing this view altogether.

**Comment by Tobias-Hunter** [comment]
There's a few items that we highlighted in an earlier discussion which I'll look to cover in the next iteration of the transaction view designs, see below:

- Incorporate the ability to submit a file for the arguments/transient data in the transaction view. 
- Iterate on how the transaction view is launched either by right clicking on a specific smart contract, transaction name or [another appropriate method here] or all.
- General tidy the whole transaction view up and provide hi-fi guidance where necessary. 

Hopefully this is everything that we chatted about @erin-hughes, if not feel free to edit this comment! 🙂

**Comment by erin-hughes** [comment]
Below are screenshots of our shiny new designs. The plan is to introduce a [content switcher component](https://www.carbondesignsystem.com/components/content-switcher/usage/) that will allow users to switch between inputting their transaction arguments manually and using transaction data files.

![image](https://user-images.githubusercontent.com/35696678/95973440-914b6a80-0e0b-11eb-900d-0df3adf6a925.png)


![image](https://user-images.githubusercontent.com/35696678/95973494-a3c5a400-0e0b-11eb-80d0-e5f8cfdbc983.png)

![image](https://user-images.githubusercontent.com/35696678/95973932-323a2580-0e0c-11eb-8138-de12c1cf40c6.png)

**Comment by erin-hughes** [comment]
Handover comments for @james-wallis:

I've pushed what I've got so far to [this branch on my fork](https://github.com/erin-hughes/blockchain-vscode-extension/tree/update-txn-view-2639). Currently I've added the content switcher component mentioned above, and I've implemented the associate/dissociate transaction data directory functionality, so you won't have to worry about that!

We've not got any tutorials on transaction data files, but if you want/need info then there's a section [in our readme](https://github.com/IBM-Blockchain/blockchain-vscode-extension#using-transaction-data-files-to-submit-a-transaction) that explains how to use them.

The bits left to do are
- Submit a transaction using the selected transaction from the txdata file - this should just be a case of writing a button handler in the UI to send a message to VS Code to submit the transaction, I've done that for submitting transactions manually [here](https://github.com/IBM-Blockchain/blockchain-vscode-extension/blob/v2/packages/blockchain-ui/src/components/elements/TransactionForm/TransactionForm.tsx#L89). You'll need to update the submitTransaction command as well.
- Update the output panel to match the new designs
- Overall styling to match the new designs
- Tests (sorry)

I've tried to leave TODOs at the sections that you'll need to make changes to. If you've got any questions let me know!
